src/go/rpk/pkg/cli/cmd/generate/grafana.go

```diff
@@ -28,10 +28,10 @@ import (
 
 	dto "github.com/prometheus/client_model/go"
 	"github.com/prometheus/common/expfmt"
-	"github.com/redpanda-data/redpanda/src/go/rpk/pkg/cli"
 	"github.com/redpanda-data/redpanda/src/go/rpk/pkg/cli/cmd/generate/graf"
 	"github.com/redpanda-data/redpanda/src/go/rpk/pkg/httpapi"
 	"github.com/redpanda-data/redpanda/src/go/rpk/pkg/out"
+	"github.com/sirupsen/logrus"
 	log "github.com/sirupsen/logrus"
```

Comment:
Well, it got deleted in a few commits afterward.

Reply:
I plan to keep this here since it's removed in a follow up commit (that entirely removes logrus), and the point of this commit is to move things from log_formatter.go into the two places it's used, without any changes.